crates/bevy_gilrs/src/gilrs_system.rs

```diff
@@ -0,0 +1,187 @@
+use crate::converter::{convert_axis, convert_button, convert_gamepad_id};
+use bevy_app::Events;
+use bevy_ecs::{Res, ResMut};
+use bevy_input::prelude::*;
+use gilrs::{Button, EventType, Gilrs};
+use std::sync::{Arc, Mutex};
+
+struct GilrsSendWrapper {
```

Comment:
Lets add a `// TODO: remove this if/when bevy_ecs supports thread local resources`, just so we don't forget.

Reply:
I also have a feeling that this will break down on certain platforms, but its a decent hack until we can sort out thread local resources.

Reply:
Added